prow/cmd/deck/job_history.go

```diff
@@ -310,9 +310,11 @@ func getBuildData(bucket storageBucket, dir string) (buildData, error) {
 	}
 	if finished.Timestamp != 0 {
 		b.Duration = time.Unix(finished.Timestamp, 0).Sub(b.Started)
-	}
-	if finished.Result != "" {
-		b.Result = finished.Result
+		if finished.Passed {
+			b.Result = "SUCCESS"
```

Comment:
Can we get these as typed constants please

Reply:
Done